### PR TITLE
APIGOV-21697: Replace dependency "listr" with "listr2"

### PR DESCRIPTION
**Pull request by jquick-axway**
JIRA:
https://jira.axway.com/browse/APIGOV-21697

### CODE REVIEW

File: packages/axway-cli-pm/src/utils.js [hunk 1]
```diff
@@ -49,7 +49,6 @@ export class ListrTextRenderer {
 export async function runListr({ console, json, tasks }) {
 	await (new Listr(tasks, {
 		concurrent: 10,
-		console,
```
Comment (cb1kenobi) [inline packages/axway-cli-pm/src/utils.js:52]: Ha, good catch! `console` was never used!

File: packages/axway-cli-pm/src/utils.js [hunk 2]
```diff
@@ -49,7 +49,6 @@ export class ListrTextRenderer {
 export async function runListr({ console, json, tasks }) {
```
Comment (cb1kenobi) [inline packages/axway-cli-pm/src/utils.js:49]: `console` is no longer used and can be removed from the args and jsdoc as well as remove `console` from being passed in from `purge.js`, `update.js`, `install.js`, and `uninstall.js`.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: packages/axway-cli-pm/src/utils.js (hunk 2 above)
Comment (jquick-axway) [reply packages/axway-cli-pm/src/utils.js:49]: Sure. I'll go remove it now.

File: packages/axway-cli-pm/src/utils.js (hunk 1 above)
Comment (jquick-axway) [reply packages/axway-cli-pm/src/utils.js:52]: Oh hold on. It looks like `console` was intended to be used by your custom `ListrTextRenderer` type, right?

Comment (jquick-axway) [reply packages/axway-cli-pm/src/utils.js]: I re-adding `console` arg since custom `ListrTextRenderer` needs it.

**Review by cb1kenobi — APPROVED** [review]
APPROVED!

**Pull request merged**